pefile.py

```diff
@@ -2713,7 +2717,6 @@ def get_symbol_name(symbol, string_table, string_table_size):
 
                 return name
 
-            self.symbols = []
```

Comment:
This value is reference elsewhere and can cause _AttributeError_ if it's not defined.